app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -21,6 +21,7 @@ def show
     end
 
     def create
+      mfa_selection_attempt_count('otp')
```

Comment:
I think since these values are "known", it makes more sense to use symbol values rather than strings.

(same comment across the board for the hash keys)

```suggestion
      mfa_selection_attempt_count(:otp)
```

Related: https://fullstackheroes.com/tutorials/ruby/symbols-vs-strings/

Reply:
That makes sense. Thanks!